lib/SensorFusion/src/App.h

```diff
@@ -86,13 +86,33 @@ class App
      */
     void loop();
 
+    /**
+     * Publish Position calculated by Sensor Fusion via MQTT.
+     * It will only be published if a new End Line has been detected.
+     */
+    void publishSensorFusionPosition();
+
+    /**
+     * Process the Receiving of New Sensor Data via SerialMuxProt
+     * 
+     * @param[in] newData New Sensor Data.
+     */
+    void processNewSensorData(const SensorData newData);

```

Comment:
You could pass a const reference to an object instead.  `const SensorData &newData`
Or am I wrong?